settings.py

```diff
@@ -2,36 +2,30 @@
 # -*- coding: utf-8 -*-
 
 from wger.settings_global import *
-from decouple import config
-import dj_database_url
-import os
 
 # Use 'DEBUG = True' to get more details for server errors
 DEBUG = True
 TEMPLATES[0]['OPTIONS']['debug'] = True
 
 ADMINS = (
-    (os.environ.get('ADMIN_NAME'), os.environ.get('ADMIN_EMAIL')),
+    ('Your name', 'your_email@example.com'),
```

Comment:
Revert this line. You need to set the appropriate environment variables and use those rather than hard-coding them here.